src/commands/makersuite.rs

```diff
@@ -49,26 +49,26 @@ impl CommandTrait for GoogleGemini {
         }
 
         if let Some(message_image) =
-            telegram_utils::get_message_or_reply_image(&ctx.message, ctx.client_id).await
+            telegram_utils::get_message_or_reply_attachment(&ctx.message, ctx.client_id).await
         {
-            if message_image.file.expected_size > 4 * MEBIBYTE {
-                return Err(CommandError::Custom("the image cannot be larger than 4 MiB.".into()));
+            if message_image.filesize() > 20 * MEBIBYTE {
+                return Err(CommandError::Custom("the media file cannot be larger than 20 MiB.".into()));
```

Comment:
you can just call it a “file”
```suggestion
                return Err(CommandError::Custom("the file cannot be larger than 20 MiB.".into()));
```